bundles/net.tourbook.cloud/src/net/tourbook/cloud/dropbox/DropboxClient.java

```diff
@@ -216,8 +217,10 @@ public static final String getValidTokens() {
          return UI.EMPTY_STRING;
       }
 
-      if (!OAuth2Utils.isAccessTokenExpired(_prefStore.getLong(Preferences.DROPBOX_ACCESSTOKEN_ISSUE_DATETIME) + _prefStore.getInt(
-            Preferences.DROPBOX_ACCESSTOKEN_EXPIRES_IN) * 1000)) {
+      if (OAuth2Utils.isAccessTokenValid(
+            _prefStore.getLong(Preferences.DROPBOX_ACCESSTOKEN_ISSUE_DATETIME) +
+                  _prefStore.getInt(Preferences.DROPBOX_ACCESSTOKEN_EXPIRES_IN) * 1000)) {
+
```

Comment:
This code is now better readable

Reply:
And also because I renamed the function and removed the double negation :smile: